Hyperparameter_optimization/README.md

```diff
@@ -1,7 +1,7 @@
 # Hyperparameter optimization
 Optimize hyperparameters for 12 hours, over 6 datasets in the `data` directory for a model with Naive Bayes classifier (B), tfidf feature extraction (T), certainty sampling (max) query strategy (C), double balance strategy (D):
 
-`mpirun -n 2 asreview hyper-active -m nb -b double -e tfidf -q max -t 12:00:00 --mpi`
+`mpirun -n 2 asreview hyper-active -m nb -b double -e tfidf -q max --mpi`
```

Comment:
Can you explain this one?

Reply:
There is no "-t" flag for the asreview hyper-active entry-point, so I removed it from the command (it will give errors).